src/main/java/uk/gov/hmcts/reform/prl/controllers/CallbackController.java

```diff
@@ -1010,5 +1010,27 @@ public AboutToStartOrSubmitCallbackResponse populateChildInformation(@RequestHea
         }
         return AboutToStartOrSubmitCallbackResponse.builder().data(caseDataUpdated).build();
     }
+
+    @PostMapping(path = "/update-refuge-confidential-details", consumes = APPLICATION_JSON, produces = APPLICATION_JSON)
```

Comment:
Can this name be generic, so in future any other business logic needs to be added here will be taken care. Can this be changed to update-party-details and then the service with in it could be specific to refuge?

Reply:
need a seperate controller for this one. with specific service, can be picked up with other tickets.